specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2021-06-01/iotcentral.json

```diff
@@ -533,6 +533,35 @@
       },
       "readOnly": true
     },
+    "ManagedIdentity": {
+      "description": "The managed identity for IoT Central application.",
+      "title": "Managed Identity",
+      "type": "object",
+      "properties": {
```

Comment:
nit: Lets be a bit more descriptive on these properties. 

`principalId` The AD principal ID. Also known as the objectId

Reply:
sure, updated!

Reply:
FYI, ManagementIdentify can also be refer to common definition: https://github.com/Azure/azure-rest-api-specs/blob/0a2fe04ff85f61229fed5602ccbe3f467d508a31/specification/common-types/resource-management/v3/managedidentity.json#L49 
You don't have to define your own one